docs/NEW_GUIDE_DRAFT.en.md

```diff
@@ -0,0 +1,68 @@
+`NEAR CLI Core commands`
+login
+create-account
+    - implicit
+    - subaccount
+<!-- account Id is entered in this step -->
+manage-account
+    - keys
+        - view
+        - add
+        - delete
+        - nonce
+    - local-keys
+        <!-- Needs to be designed (with multikey management) -->
+    - contract
+        - deploy
+        - dev-deploy
+        - state
+        - wasm-hash
+        - view-call
+        - change-call
+    - state
+    - delete
+transfer
```

Comment:
To be honest, I don't like the idea of having `login`, `transfer`, on the top level since they don't match the idea of extensions.

Reply:
> where they should be then?

Sorry, I have not put much thought into it yet. Maybe we should put `transfer` under `manage-account` -> `balance` -> `transfer`.

`login` functionality is currently tightly coupled with the NEAR Wallet. There are other ways to get access to your account:

* import account using a seed phrase (we can replicate the Wallet flow of recovering the master key, and then generate and add a new access key that will be stored locally [we should never store the master key pair or seed phrase])
* import account using the key in Ledger (here we have two options: (1) follow the previous approach of adding a new access key [see above] (2) just record that this key is available through Ledger, so near-cli-rs then can hint user to connect Ledger device and follow instructions when one needs to sign a transaction)
* import account using a plaintext private key (I have heard this to be a common feature request for Wallet)
* we might also consider adding flows to create accounts from a linkdrop

Thus, instead of `login` we might want to have a whole top-level command like `import-account`

> And how it contradicts the extensions idea?

top-level commands in my mind should denote some group of actions while `login` and `transfer` are direct actions (this is prone to having more direct actions on the top-level, and then we will be back to dozen options on the top-level)

Reply:
Great suggestions about account import options. So `login` becomes `import-account` -> `from-near-wallet`.
I'm adding other options as well.

I agree that `transfer/send` is out of place here.
But we do not have `balance` for `manage-account` -> `balance` -> `transfer` in the current design.
What do you think about  `mange-account` -> `transfer`?